src/raftstore/store/peer.rs

```diff
@@ -1786,6 +1872,17 @@ impl Peer {
         send_msg.set_from_peer(from_peer);
         send_msg.set_to_peer(to_peer);
 
+        // Backward compatibility for `sync-log`, it has been moved into `ProposalContext`.
+        // TODO: remove it in the next major release.
+        for e in msg.mut_entries().iter_mut() {
```

Comment:
why do we still need to iterate the entries and set sync-log? Is this flag not set in Propose?

Reply:
`sync-log` has been moved into `ProposalContext`, it's for 2.0.1 sends to 2.0.0, I can remove it if you want.